libsubid/subid.h

```diff
@@ -21,6 +21,20 @@ enum subid_status {
 	SUBID_STATUS_ERROR = 3,
 };
 
+/*
+ * libsubid_init: initialize libsubid
+ *
+ * @progname: Name to display as program.  If NULL, then "(libsubid)" will be
```

Comment:
I would mention pointer is kept (not memory copy). Thus caller should not release memory.

Reply:
Hm, actually I'm doing an strdup now instead.  Prefer not to risk segfaults due to an accidental caller garbage collection.

Reply:
Ok.